### DISCUSSION TITLE
Stumpy library with dask client

### DISCUSSION THREAD

**Discussion by adityabhandwalkar** [dask/distributed#7466] [Q&A]
Hello Community,

I've been working with the a library called [stumpy](https://stumpy.readthedocs.io/en/latest/api.html) which also supports dask. Now in this library there is module called [ostinatoed](https://stumpy.readthedocs.io/en/latest/api.html#ostinatoed) which utilizes the Ray Distributed client for  distributed cluster. Can anyone help me with this coding implementation. for the understanding  there is one version which works without [dask](https://stumpy.readthedocs.io/en/latest/api.html#ostinato)

Thank you

**Comment by jrbourbeau** [comment]
Thanks @adityabhandwalkar. Maybe @seanlaw has insight here

**Reply by seanlaw** [reply]
@jrbourbeau I think @adityabhandwalkar question is really "How do I set up a distributed Dask cluster?". After that, using Dask with `stumpy.ostinatoed` would be as simple as passing the Dask client into the function and STUMPY would handle the rest:

```
from dask.distributed import Client

if __name__ == "__main__":
    with Client() as dask_client:
        stumpy.ostinatoed(
            dask_client,
            [np.array([584., -11., 23., 79., 1001., 0., 19.]),
             np.array([600., -10., 23., 17.]),
             np.array([  1.,   9.,  6.,  0.])],
            m=3)
```

The missing part for @adityabhandwalkar is "How to set up a distributed Dask cluster", which unfortunately is beyond the scope of support for STUMPY. I had already pointed @adityabhandwalkar to the relevant Dask documentation.

**Reply by jrbourbeau** [reply]
Ah, I see. Thanks for clarifying @seanlaw! 

@adityabhandwalkar I'd recommend taking a look at https://docs.dask.org/en/stable/deploying.html (it sounds like @seanlaw already pointed you there). If you have a specific question about setting up clusters, I'd recommend posting on the Dask Discourse https://dask.discourse.group/ (this is where usage questions go). Hopefully that helps
